Sources/ProjectSpec/Scheme.swift

```diff
@@ -253,8 +253,8 @@ extension Scheme.Run: JSONObjectConvertible {
     public init(jsonDictionary: JSONDictionary) throws {
         config = jsonDictionary.json(atKeyPath: "config")
         commandLineArguments = jsonDictionary.json(atKeyPath: "commandLineArguments") ?? [:]
-        preActions = try jsonDictionary.json(atKeyPath: "preActions")?.map(Scheme.ExecutionAction.init) ?? []
-        postActions = try jsonDictionary.json(atKeyPath: "postActions")?.map(Scheme.ExecutionAction.init) ?? []
+        preActions = jsonDictionary.json(atKeyPath: "preActions") ?? []
+        postActions = jsonDictionary.json(atKeyPath: "postActions") ?? []
```

Comment:
👍 Did something change? I'm trying to recall why I would have added the `map`s if they're not necessary.

Reply:
Nothing changed. I just noticed these, and removed the unnecessary map. If you can think of a reason why they were there, let me know! 😀